site/layouts/partials/docs-navbar.html

```diff
@@ -24,9 +24,9 @@
     </div>
 
     <div class="offcanvas-lg offcanvas-end flex-grow-1" tabindex="-1" id="bdNavbar" aria-labelledby="bdNavbarOffcanvasLabel" data-bs-scroll="true">
-      <div class="offcanvas-header px-4 pb-0">
+      <div class="offcanvas-header px-4 pb-0" data-bs-theme="dark">
```

Comment:
I've put `data-bs-theme="dark"` at this level in the DOM because otherwise, the dropdowns would be black. This is linked to something that is still not clear in our documentation (and to me) regarding dark variants vs. contextual dark mode.